content-management-system/src/scripts/configuration.sql

```diff
@@ -0,0 +1,88 @@
+-- Create Division table
+CREATE TABLE Division (
+                          divisionId INT AUTO_INCREMENT,
+                          name VARCHAR(255) UNIQUE NOT NULL,
```

Comment:
No need to add unique keyword here. As we are adding uniqe constraint at the bottom explicitly.

Reply:
changes done for all create table sql scripts